ceremony.md

```diff
@@ -0,0 +1,16 @@
+# Ceremony
+
+An out-of-band provisioning of cryptographic keys is required to bootstrap cryptographic security among participants of re-vault. Countermeasures should be in place to minimize the risks associated with physical, network, and social engineering attacks that aim to compromise this process. Ideally this ceremony would not be too cumbersome for the humans involved and the scope for human error should be minimized.
+
+## Objectives
+
+Prior to the ceremony, it is assumed that the architectural configuration has been specified, the devices and machines have been aquired and the appropriate software has been installed (including blockchain download for all full nodes). 
+
+1. Each non-manager gets the hidden service details (.onion) for the set of co-signing servers that are each operated by a manager.
```

Comment:
This is the other way around :sweat_smile:

Reply:
I thought: managers generate .onion details prior to ceremony. During the ceremony these are shared with non-managers?

Reply:
Yeah, that's what i said:
```suggestion
1. Each manager gets the hidden service details (.onion) for the set of co-signing servers that are operated by each non-manager.
```

Reply:
I think I didn't state it clearly but I think we agree. Just to clarify: 

Managers generate .onion details when they set up the co-signing server. They will manage this machine and restrict physical access to it. During the ceremony the .onion address is shared with non-managers. Non-managers communicate with the co-signing servers but don't know where they are and cannot physically access them.